app/internal/event.py

```diff
@@ -84,3 +90,23 @@ def get_messages(session: Session,
         messages.append(f'Same event happened {weeks_diff} weeks before too. '
                         f'Want to create another one {weeks_diff} after too?')
     return messages
+
+
+def get_location_coordinates(
+        address: str,
+        timeout: float = LOCATION_TIMEOUT
+        ) -> NamedTuple:
```

Comment:
The type here should be `Location`

Reply:
Thanks. Created a custom type class with the same name.